contracts/Proxy.sol

```diff
@@ -1,45 +1,15 @@
-// VERSION 1
-// pragma solidity 0.4.9;
-// import "../libraries/Owned.sol";
-// contract Proxy is Owned {
-//   event Forwarded (address indexed destination, uint value, bytes data );
+pragma solidity 0.4.15;
+import "./libs/Controlled.sol";
 
-//   function () payable {}
 
-//   function forward(address destination, uint value, bytes data) onlyOwner {
-//     if (!destination.call.value(value)(data)) { throw; }
-//     Forwarded(destination, value, data);
-//   }
-// }
+contract Proxy is Controlled {
+    event LogForwarded (address indexed destination, uint value, bytes data);
```

Comment:
@oed was it necessary to rename these? There may be code depending on these names. Did it come from the security review?